src/DebuggerForm.cpp

```diff
@@ -1149,7 +1152,8 @@ void DebuggerForm::toggleBreakpoint(int addr)
 		// get slot
 		auto [ps, ss, seg] = addressSlot(addr);
 		// create command
-		cmd = Breakpoints::createSetCommand(Breakpoint::BREAKPOINT, addr, ps, ss, seg);
+		cmd = Breakpoints::createSetCommand(Breakpoint::BREAKPOINT, AddressRange{addr, {}},
```

Comment:
detail: this is a pattern that I've seen repeated a few times: constructing an AddressRange with an explicit empty 2nd parameter. It would we nice if we didn't have to write this 2nd parameter.

To obtain this we can give the AddressRange struct (in DebuggerData.h) two constructors:

    struct AddressRange {
        explicit AddressRange(uint16_t addr) : start(addr) {}
        AddressRange(uint16_t b, uint16_t e) : start(b), end(e) {}
        // only add this 3rd version when really needed
        AddressRange(uint16_t b, std::optional<uint16_t> e) : start(b), end(e) {}
        ...
    };

It's only a small improvement, but it is recurring in several places. Feel free to apply this tweak or not.